src/gtc/dace/expansion.py

```diff
@@ -926,6 +926,268 @@ def add_nodes_and_edges(self):
                 tasklet.in_connectors[edge.dst_conn] = dace.pointer(array.dtype)
 
 
+class BlockVerticalLoopExpander(NaiveVerticalLoopExpander):
+
+    default_tile_sizes = (64, 8)
```

Comment:
Probably better capitalized to indicate this is a constant. Could also be a [`ClassVar`](https://docs.python.org/3/library/typing.html#typing.ClassVar).